topics/epigenetics/slides/introduction_atac.html

```diff
@@ -0,0 +1,121 @@
+---
+layout: introduction_slides
+logo: "GTN"
+video: true
+
+title: "ATAC-Seq data analysis"
+type: "introduction"
+questions:
+  - What is ATAC-Seq?
+  - What are the quality parameters to check for each dataset?
+  - How to analyse ATAC-Seq data?
+objectives:
+  - Understand ATAC-Seq
+  - Quality Parameters for ATAC-Seq
+  - Understand Peak calling for ATAC-Seq
+requirements:
+time_estimation: "30min"
+key_points:
+  - Run quality control on every sequencing dataset before any other analyses
+  - Choose QC parameters carefully
+  - Re-run FastQC to check the impact of the quality control
+contributors:
+  - heylf
+
+---
+
+# What is ATAC sequencing?
+
+---
+
+### Where my data comes from?
+
+![](../images/atac-seq/atac.jpg)
+
+<small>
+[*Buenrostro et al. 2013 Nat Methods*](https://doi.org/10.1002/0471142727.mb2129s109)
+</small>
+
+- Usage of hyperactive Tn5 transposase to insert sequencing adapters into open chromatin regions.
+- After adapter attachment the DNA is sheared by the Transposase itself.
+
+???
+
+It requires sequence-specific excision of a locus containing 19 base-pair inverted repeats.
+
+You have reads without nucleosomes (histones) = open chromatin regions, and reads with these complexes which will be filtered out because of their length.
```

Comment:
They are not filtered out. If the length is too big (>800bp) it is not correctly amplified by PCR and/or not efficiently sequenced by illumina sequencers but we for sure keep the pairs with 1 or 2 nucleosomes inside. That's why it is so important to build coverage around the insertion sites and not fully between the mates of the pairs: if the pairs are larger than 170bp you don't know if there was a nucleosome between them or if everyting was 'naked' but you know that where the Tn5 cuts there was no nucleaosome.

Reply:
check